### PR TITLE
update prisma

### PR DESCRIPTION
**Pull request by kanolato**
Signed-off-by: Miguel Angel Medina Mondragon <kanolato@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ x] Chart Version bumped
- [ x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prisma/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: prisma
-version: 1.1.0
-appVersion: 1.20
+version: 1.1.1
```
Comment (giacope) [inline stable/prisma/Chart.yaml]: Can you make this a minor bump instead of a patch?

**Review by giacope — CHANGES_REQUESTED** [review]

File: stable/prisma/Chart.yaml (hunk 1 above)
Comment (kanolato) [reply stable/prisma/Chart.yaml]: of course

Comment (kanolato) [reply stable/prisma/Chart.yaml]: @giacomoguiulfo  done!

**Pull request merged**